### ISSUE TITLE
Allow icons to have title text for acccessibility

### ISSUE THREAD

**Issue by imcbride** [middlebury/frontend#177]
When icons are the only element inside a link, the link will not contain any text and will fail WCAG 2.4.4 Link Purpose (In Context) Level A. The purpose of a [`<title>` element inside the `<svg>` element](https://developer.mozilla.org/en-US/docs/Web/SVG/Element/title) is to provide a short description of the graphic and will not be displayed by the browser other than, possibly, as a tooltip.
